tests/integration/src/test/java/org/apache/pulsar/tests/integration/loadbalance/ExtensibleLoadManagerTest.java

```diff
@@ -405,7 +405,7 @@ public void testIsolationPolicy() throws Exception {
                         admin.lookups().lookupTopicAsync(topic).get(5, TimeUnit.SECONDS);
                     } catch (Exception ex) {
                         log.error("Failed to lookup topic: ", ex);
```

Comment:
It seems that the lookup error could be `TimeoutException` before retrying until the error becomes "Service Unavailable". Is it expected?

Reply:
This PR should not cause the `TimeoutException`, I think we should merge this PR first, then I will try to find the root cause and push another PR to fix it.